shared-libs/rules-engine/test/pouchdb-provider.spec.js

```diff
@@ -68,9 +68,10 @@ describe('pouchdb provider', () => {
 
   describe('allTasks', () => {
     it('for owner', async () => expect(await pouchdbProvider(db).allTasks('owner')).excludingEvery('_rev')
-      .to.deep.eq([headlessTask, taskOwnedByChtContact]));
+      .to.have.deep.members([headlessTask, taskOwnedByChtContact, taskRequestedByChtContact]));
+
     it('for requester', async () => expect(await pouchdbProvider(db).allTasks('requester')).excludingEvery('_rev')
-      .to.deep.eq([headlessTask, taskRequestedByChtContact]));
+      .to.have.deep.members([headlessTask, taskRequestedByChtContact]));
```

Comment:
I wouldn't have expected this to need to change as the `requester` branch of the view hasn't been updated.

Reply:
It was just for consistency.

`to.have.deep.members` is the correct assertion to use, since we don't care about the array element order.